tests/scripts/run_performance.sh

```diff
@@ -36,7 +36,7 @@ run_perf_models_llm_javelin() {
     local tt_arch=$1
     local test_marker=$2
 
-    env pytest models/demos/falcon7b/tests -m $test_marker
+    env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon7b/tests -m $test_marker
 
     if [ "$tt_arch" == "wormhole_b0" ]; then
         env pytest models/demos/mamba/tests -m $test_marker
```

Comment:
Have you tested `WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml` with the [t3000](https://github.com/tenstorrent/tt-metal/tree/main/models/demos/t3000/falcon7b) and [wormhole](https://github.com/tenstorrent/tt-metal/tree/main/models/demos/wormhole/falcon7b) demos? If there is no drop in e2e perf when setting [perf_mode](https://github.com/tenstorrent/tt-metal/blob/main/models/demos/t3000/falcon7b/demo_t3000.py#L10)=True, could you please update the command in the readme to use this?